src/commands/mapsheet-coverage/mapsheet.coverage.ts

```diff
@@ -0,0 +1,224 @@
+          title: collection.title,
+          layerSizeBytes: Math.round(JSON.stringify(captureArea.geometry).length),
+        },
+        'Processing',
+      );
+
+      // GeoJSON over about 8 decimal places starts running into floating point math errors
+      truncateGeoJson(captureArea);
+
+      // Determine if this layer has any additional information to the existing layers
+      const diff = pc.difference(captureArea.geometry.coordinates as pc.MultiPolygon, layersCombined);
+
+      // Layer has no information included in the output
+      if (diff.length === 0) {
+        logger.warn({ layer: layer.name, location: targetCaptureAreaUrl.href }, 'FullyCovered');
+        await fsa.write(fsa.join(OutputPath, `remove-${layer.name}.geojson`), JSON.stringify(captureArea));
+        continue;
+      }
+
+      // Extract all the links from the collection and assign them to each mapsheet
+      // This assumes that tiles are in the format "./BX01_1000_0101.json"
+      for (const link of collection.links) {
+        if (link.rel !== 'item') continue;
+        const url = new URL(link.href, targetCollection);
+        const fileName = basename(url.pathname);
+
+        const ms = MapSheet.getMapTileIndex(fileName);
+        if (ms == null) throw new Error('Unable to extract mapsheet from ' + url.href);
+
+        // Limit the output to only the requested mapsheet
+        if (args.mapSheet && args.mapSheet !== ms.mapSheet) continue;
+
+        const existing = mapSheets.get(ms.mapSheet) ?? [];
+        // TODO this is not the safest way of getting access to the tiff, it would be best to load
+        existing.push(url.href.replace('.json', '.tiff'));
+        mapSheets.set(ms.mapSheet, existing);
+      }
+
+      layersCombined = pc.union(layersCombined, captureArea.geometry.coordinates as pc.MultiPolygon);
+      layersRequired.features.push({
+        type: 'Feature',
+        geometry: { type: 'MultiPolygon', coordinates: diff },
+        properties: captureArea.properties,
+      });
+      allLayers.features.push(captureArea);
+    }
+
+    // All the source layers as a single file
+    logger.info('Write:SourceFeatures');
+    await fsa.write(fsa.join(OutputPath, 'layers-source.geojson.gz'), gzipSync(JSON.stringify(allLayers)));
+
+    // A single output feature for total capture area
+    logger.info('Write:CombinedUnion');
+    await fsa.write(
+      fsa.join(OutputPath, 'layers-combined.geojson.gz'),
+      gzipSync(JSON.stringify({ type: 'Feature', geometry: { type: 'MultiPolygon', coordinates: layersCombined } })),
+    );
+
+    // Which areas of each layers are needed for the output
+    logger.info('Write:RequiredLayers');
+    await fsa.write(fsa.join(OutputPath, 'layers-required.geojson.gz'), gzipSync(JSON.stringify(layersRequired)));
+
+    // List of files to be created
+    const todo = [...mapSheets].map((m) => {
+      return { output: `${m[0]}.tiff`, input: m[1].reverse(), includeDerived: true };
```

Comment:
```suggestion
      return { output: `${m[0]}`, input: m[1].reverse(), includeDerived: true };
```

To be consistent with our other "output" files